crates/bevy_pbr/src/material.rs

```diff
@@ -741,6 +741,7 @@ pub fn queue_material_meshes<M: Material>(
             view_key |= match projection {
                 Projection::Perspective(_) => MeshPipelineKey::VIEW_PROJECTION_PERSPECTIVE,
                 Projection::Orthographic(_) => MeshPipelineKey::VIEW_PROJECTION_ORTHOGRAPHIC,
+                Projection::Custom(_) => MeshPipelineKey::VIEW_PROJECTION_NONSTANDARD,
```

Comment:
Note to others: We do appear to be correctly pushing the shader def for this already, it just isn't set anywhere by our code. Since `Camera3d` has a require component of `Projection`, presumably users would have had to manually remove that projection and add `VIEW_PROJECTION_NONSTANDARD` themselves.